spec/acceptance/cli_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper_acceptance'
+
+describe 'installation of cli only' do
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<-PUPPET
+      include pulpcore::cli
+      PUPPET
+    end
+  end
+
+  describe command("pulp config validate --location /etc/pulp/cli.toml") do
+    its(:exit_status) { is_expected.to eq 0 }
+    its(:stdout) { is_expected.to match(/valid pulp-cli config/) }
+  end
+end
+
+describe 'installation of server with cli' do
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<-PUPPET
+      include pulpcore
+      class { 'pulpcore::cli':
+        pulpcore_url => "https://#{host_inventory['fqdn']}/",
```

Comment:
Could also be written using native Puppet:
```suggestion
        pulpcore_url => "https://${facts['fqdn']}/",
```